packages/babel-helper-create-class-features-plugin/src/index.ts

```diff
@@ -44,7 +46,16 @@ export function createClassFeaturePlugin({
   manipulateOptions,
   api,
   inherits,
+  decoratorVersion,
 }: Options): PluginObject {
+  if (
+    decoratorVersion === "2021-12" ||
+    decoratorVersion === "2022-03" ||
+    decoratorVersion === "2023-01" ||
+    decoratorVersion === "2023-05"
```

Comment:
How does this option interact with `FEATURES`? For Babel 8, can we just ignore this option and rely on a feature flag?

Reply:
Good question, it should be wrapped within the `FEATURES.Decorators` check. As for Babel 8, yes we can just ignore the `decoratorVersion` and eventually remove it from the features plugin.

Reply:
The `FEATURES` is a per-file feature set that can be accessed in the plugin visitor, however `createDecoratorTransform` returns a new plugin object, at this moment we have to temporarily use `decoratorVersion` until we blend the decorator transform with other features.